tests/init_db.py

```diff
@@ -1,63 +1,24 @@
-from src import database
+from src import database, models
+from sqlalchemy import text
+import os
 
-
-def clear_tables():
-    commands = [
-        "DROP TABLE IF EXISTS users CASCADE",
-        "DROP TABLE IF EXISTS credits CASCADE",
-        "DROP TABLE IF EXISTS entrypoints CASCADE",
-        "DROP TABLE IF EXISTS contracts CASCADE",
-        "DROP TABLE IF EXISTS operations CASCADE"
-    ]
-
-    db = database.connect()
-    cur = db.cursor()
-    for command in commands:
-        cur.execute(command)
-
-    cur.close()
-    db.commit()
+def truncate_db():
+    con = database.engine.connect()
+    trans = con.begin()
+    for table in models.Base.metadata.sorted_tables:
+        con.execute(text(f'ALTER TABLE "{table.name}" DISABLE TRIGGER ALL;'))
+        con.execute(table.delete())
+        con.execute(text(f'ALTER TABLE "{table.name}" ENABLE TRIGGER ALL;'))
+    trans.commit()
```

Comment:
Interesting, what does this do exactly?

Reply:
It delete all rows in all tables instead of delete them (I don't know why I do that instead of deleting 😬 )
To delete data we need to disable some checks (like dependencies between tables) . If we don't disable this checks, it will throw errors because we try to delete a row that depends on another row in other table.